projects/dev-schematics/src/ng-add/index.ts

```diff
@@ -8,27 +8,26 @@ import {
   Tree,
 } from '@angular-devkit/schematics';
 import { getSpartacusVersion } from '../shared/utils/package-utils';
+import { Schema as DevSpartacusOptions } from './schema';
 
-export default function (options: any): Rule {
+export default function (options: DevSpartacusOptions): Rule {
   return (host: Tree, context: SchematicContext) => {
     if (!getSpartacusVersion(host)) {
       throw new SchematicsException(
         'Could not find Spartacus in package dependencies. Please install Spartacus before using this schematic.'
       );
     }
 
-    context.logger.info(options);
-
     return chain([
-      options['default-base-sites'] && JSON.parse(options['default-base-sites'])
+      options['default-base-sites'] && options['default-base-sites']
```

Comment:
Not sure why do we need to check for `options['default-base-sites']` two times?
The same goes for line 26 and 30